lib/interface/controls/card.ts

```diff
@@ -0,0 +1,27 @@
+import { type IconType } from 'react-icons'
+
+// export type ConditionTypeforCard<T> = {
+//   [K in keyof T]: K extends 'primaryColor'
+//     ? Extract<T, { variant: 'image' }>['variant'] extends 'image'
+//       ? never
+//       : T[K]
+//     : K extends 'imgUrl' | 'members' | 'subject' | 'desc'
+//       ? Extract<T, { variant: 'default' | 'horizontal' }>['variant'] extends
+//           | 'default'
+//           | 'horizontal'
+//         ? never
+//         : T[K]
+//       : T[K]
+// }
+
+export interface CardI extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
First, name this interface `CardProps` to illustrate what it is for.

Don't extend from `React.HTMLAttributes<HTMLDivElement>`; define just the props that are unique to the component. If a prop is not explicitly defined here, it should be considered unused and would show an error if an attempt is made to pass it to the `Card` component. Otherwise, users could pass props you didn't anticipate, causing problems later on.